doc/source/ray-observability/user-guides/configure-logging.md

```diff
@@ -384,10 +384,14 @@ ray.get(f.remote("A log message for a task."))
 
 ```{admonition} Caution
 :class: caution
-This is an experimental feature. It doesn't support [Ray Client](ray-client-ref) yet.
+This is an experimental feature. The semantic of the API is subject to change.
+It doesn't support [Ray Client](ray-client-ref) yet.
+
+When you use `ray.init(runtime_env={...})` with a [Ray Job Submission](jobs-quickstart) API (`ray job subimt --working-dir` or `ray job submit --runtime-env ...`), runtime_env passed to `ray.init` is ignored.
```

Comment:
```suggestion
When you use `ray.init(runtime_env={...})` with a [Ray Job Submission](jobs-quickstart) API (`ray job submit --working-dir` or `ray job submit --runtime-env ...`), runtime_env passed to `ray.init` is ignored.
```

Reply:
This is hard to parse. Are you saying that these two switches don't work when you just Ray Jobs?

Reply:
ray job ... is a job API. When you use the job API, the driver runtime env is ignored.

Reply:
Any recommended sentences?

Reply:
I changed it. Can you take a look? If there's no response by today, I will just merge a PR!